### DISCUSSION TITLE
Crash Using Models

### DISCUSSION THREAD

**Discussion by AgentHitmanFaris** [lllyasviel/Fooocus#3609] [Q&A] [answered]
[Parameters] Steps = 30 - 15
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE

D:\AIGeneration>pause
Press any key to continue . . .

Test out juggernautXL_v9.safetensors
and it's happen to crash

Using juggernautXL_v8Rundiffusion.safetensors models just fine it's the new model does not support or something else

**Comment by Rogala** [comment, marked as answer]
[I don't know where you found this version, try downloading from X or XI they definitely work and are newer.](https://civitai.com/models/133005?modelVersionId=456194)
